feature_engine/testing/ipython_notebooks/temporal_features_test.ipynb

```diff
@@ -0,0 +1,163 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import sys \n",
+    "import os, glob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sys.path.append('/Users/xehu/Desktop/Team Process Map/team-process-map/feature_engine/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from features.temporal_features import *"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import utils.summarize_chat_level_features as summary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def mean_msg_duration(df,on_column):\n",
+    "\n",
+    "    #convert timestamp column to datetime type (in minutes)\n",
+    "    df[on_column] = pd.to_datetime(df[on_column])\n",
+    "\n",
+    "    # calculate difference between consecutive timestamps\n",
+    "    df['time_diff'] = df[on_column].diff().dt.total_seconds()/60\n",
+    "    \n",
+    "    return summary.get_average(df, 'time_diff', 'avg_time')\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def stddev_msg_duration(df,on_column):\n",
+    "\n",
+    "    #convert timestamp column to datetime type (in minutes)\n",
+    "    df[on_column] = pd.to_datetime(df[on_column])\n",
+    "\n",
+    "    # calculate difference between consecutive timestamps\n",
+    "    df['time_diff'] = df[on_column].diff().dt.total_seconds()/60\n",
+    "\n",
+    "    return summary.get_stdev(df, 'time_diff', 'std_dev_time')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    0.0\n",
```

Comment:
In the test notebook, all of the SD's end up as 0's. This seems to be a bug, not a feature?